pkgs/development/tools/rust/cargo-readme/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-readme";
+  version = "3.2.0";
+
+  src = fetchFromGitHub {
+    owner = "livioribeiro";
+    repo = pname;
+    rev = "cf66017c0120ae198210ebaf58a0be6a78372974";
```

Comment:
Upstream tagged the releases till 3.1.2. Looks like that it was not done for 3.2.0.

Could you please poke upstream to do it?